src/main/static/assets/images/lightbend.svg

```diff
@@ -0,0 +1 @@
,5.59,0,0,1,3.06.9A58.76,58.76,0,0,0,539,201.57c20.52,0,33.84-11.34,33.84-33.12v-11c-8.1,10.8-19.44,19.62-36.54,19.62-22.32,0-43.74-16.74-43.74-43.56v-0.36c0-27.18,21.6-43.92,43.74-43.92,17.46,0,28.8,8.64,36.36,18.72V97.36a6.87,6.87,0,0,1,6.84-7,7,7,0,0,1,7,7V168.1c0,14.4-4.32,25.38-11.88,32.94-8.28,8.28-20.7,12.42-35.46,12.42A72.88,72.88,0,0,1,500.33,202.47Zm72.9-69.12V133c0-19.08-16.56-31.5-34.2-31.5s-32.22,12.24-32.22,31.32v0.36c0,18.72,14.94,31.68,32.22,31.68C556.66,164.86,573.22,152.08,573.22,133.36Z" style="fill:#fff"/><path d="M618.4,59a6.93,6.93,0,1,1,13.86,0v48.42c6.12-10.08,15.84-18.18,31.86-18.18,22.5,0,35.64,15.12,35.64,37.26v51.66a6.93,6.93,0,1,1-13.86,0V129.94c0-17.28-9.36-28.08-25.74-28.08-16,0-27.9,11.7-27.9,29.16v47.16a6.77,6.77,0,0,1-6.84,7,6.92,6.92,0,0,1-7-7V59Z" style="fill:#fff"/><path d="M732.52,159.64V103.48H725a6.31,6.31,0,0,1-6.12-6.12A6.18,6.18,0,0,1,725,91.24h7.56v-22a6.88,6.88,0,0,1,6.84-7,7.15,7.15,0,0,1,7,7v22H770.5a6.34,6.34,0,0,1,6.3,6.12,6.22,6.22,0,0,1-6.3,6.12H746.38v54.36c0,11.34,6.3,15.48,15.66,15.48,4.86,0,7.38-1.26,8.46-1.26a6,6,0,0,1,5.94,5.94,5.79,5.79,0,0,1-4,5.4,35,35,0,0,1-13.68,2.52C743.86,185.92,732.52,178.54,732.52,159.64Z" style="fill:#fff"/><path d="M806.5,59a6.93,6.93,0,1,1,13.86,0V110c7.56-11.16,18.54-20.7,35.64-20.7,22.32,0,44.46,17.64,44.46,48.24v0.36c0,30.42-22,48.42-44.46,48.42-17.28,0-28.44-9.36-35.64-19.8v11.7a6.88,6.88,0,0,1-6.84,7,7,7,0,0,1-7-7V59Zm79.74,79v-0.36c0-22-15.12-36-32.76-36-17.28,0-33.66,14.58-33.66,35.82v0.36c0,21.6,16.38,36,33.66,36C871.48,173.86,886.24,160.54,886.24,138Z" style="fill:#fff"/><path d="M964.9,186.46c-25.56,0-46.44-19.62-46.44-48.42v-0.36c0-26.82,18.9-48.42,44.64-48.42,27.54,0,43.2,22.5,43.2,47.16a6.45,6.45,0,0,1-6.48,6.48H932.5c2,20.16,16.2,31.5,32.76,31.5,11.52,0,20-4.5,26.82-10.8a6,6,0,0,1,10.08,4.32,6,6,0,0,1-2.16,4.5C991.17,180.88,980.74,186.46,964.9,186.46Zm27.54-53.82C991,115.72,981.28,101,962.74,101c-16.2,0-28.44,13.5-30.24,31.68h59.94Z" style="fill:#fff"/><path d="M1029.34,97.36a7,7,0,0,1,6.84-7,6.92,6.92,0,0,1,7,7v10.08c6.12-10.08,15.84-18.18,31.86-18.18,22.5,0,35.64,15.12,35.64,37.26v51.66a6.93,6.93,0,1,1-13.86,0V129.94c0-17.28-9.36-28.08-25.74-28.08-16,0-27.9,11.7-27.9,29.16v47.16a6.77,6.77,0,0,1-6.84,7,6.92,6.92,0,0,1-7-7V97.36Z" style="fill:#fff"/><path d="M1226.61,178.17a6.93,6.93,0,1,1-13.86,0v-12.6c-7.56,11.16-18.54,20.7-35.64,20.7-22.32,0-44.46-17.64-44.46-48.24v-0.36c0-30.42,22.14-48.42,44.46-48.42,17.28,0,28.44,9.36,35.64,19.8V59a6.77,6.77,0,0,1,6.84-7,6.92,6.92,0,0,1,7,7V178.17Zm-79.74-40.68v0.36c0,22,15.3,36,32.76,36,17.28,0,33.66-14.58,33.66-36V137.5c0-21.42-16.38-35.82-33.66-35.82C1161.64,101.68,1146.88,115,1146.88,137.5Z" style="fill:#fff"/></g><g id="icon"><path d="M1,195v56a10,10,0,0,0,10,10H291a10,10,0,0,0,10-10V195a557.85,557.85,0,0,1-150,20A557.85,557.85,0,0,1,1,195Z" style="fill:#fff"/><path d="M291,1H11A10,10,0,0,0,1,11V176a539.94,539.94,0,0,0,150,21,539.94,539.94,0,0,0,150-21V11A10,10,0,0,0,291,1Z" style="fill:#fff"/></g></svg>
```

Comment:
You can view the logo here if you want to make sure it's ok:

https://github.com/jroper/microprofile-site/blob/6a1c078a812bf74a7ddb607aa19194835b130dd1/src/main/static/assets/images/lightbend.svg